firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionModel.swift

```diff
@@ -136,5 +139,8 @@ class TrackingProtectionModel {
     }
 
     func clearCookiesAndSiteData() {
+        _ = CookiesClearable().clear()
+        _ = SiteDataClearable().clear()
+        selectedTab?.webView?.reload()
```

Comment:
are we able to write tests here?

Reply:
Do you have any suggestions here? or more exactly what should I test?

Reply:
I was thinking we can test the `clearCookiesAndSiteData` method in that we confirm that the data is clear or that we call the reload method. However, I feel that this logic is living in the model where it makes more sense to belong elsewhere like a `TrackingProtectionManager`.